.devcontainer/denv.sh

```diff
@@ -140,6 +140,28 @@ fi
 env="${env}\nCRTOOL=${CRTOOL}"
 [[ -n ${CRTOOL_DL} ]] && cr8="${cr8}\n${CRTOOL_DL}"
 ##############################
+CHR_VER="$(findVer 'set(laterChrome_VER' SWATFramework/CMakeLists.txt */defaults.txt)"
```

Comment:
the script looks for `laterChrome_VER` and `laterFirefox_VER` in `SWATFramework/CMakeLists.txt` and defaults.txt...